programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -95,8 +95,9 @@ static void on_ziti_event(ziti_context ztx, const ziti_event_t *event) {
 }
 
 extern dns_manager *get_dnsmasq_manager(const char* path);
+extern dns_manager *get_internal_dns();
```

Comment:
"internal"?  I hope to be able to use this in Apple PacketTunnelProvider at some point.  Would a different name make sense?  "default" or "provided" or "stock" or some such?

Reply:
internal just means "in-process" you can use it by letting DNS packets go through lwip

Reply:
I think I need to move it into tunneler lib